src/index.html

```diff
@@ -7,6 +7,511 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#page-top">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__row">
+        <section class="section" data-qa="card-hover">
```

Comment:
Set correct attribute
```suggestion
        <section class="section" data-qa="card">
```